docs/logicapps-ai-course/02_build_conversational_agents/07-deploying-agents-to-ms-teams.md

```diff
@@ -0,0 +1,305 @@
+--- 
+title: 07 - Deploy your agents to Microsoft Teams
+description: Learn how to deploy Logic Apps conversational agents to Microsoft Teams to enable interacting with agents through the Teams chat.
+ms.service: logic-apps
+ms.topic: tutorial
+ms.date: 11/12/2025
+author: brbenn
+ms.author: brbenn
+---
+
+# Module 06 - Deploy your agents to Microsoft Teams
+
+This module demonstrates how to deploy Logic Apps conversational agents to Microsoft Teams, enabling users to interact with intelligent agents directly through Teams chat interface.
+
+When finished with this module, you'll have gained the following knowledge:
+
+- **Teams Integration Architecture**: Understanding the three-component architecture involving Microsoft Teams, Azure Bot Service, and Logic Apps
+- **Azure Bot Service Configuration**: How to create and configure Azure Bot Service as a proxy between Teams and Logic Apps
+- **Custom Proxy Implementation**: Building a web application that implements the bot activity protocol to bridge Teams and Logic Apps
+- **Teams App Manifest Creation**: Creating and configuring Teams app manifests for sideloading custom agents
+- **End-to-End Deployment**: Complete workflow from local development to Teams production deployment
+
+## Architecture Overview
+
+The integration between Microsoft Teams and Logic Apps involves three key components:
+
+1. **Microsoft Teams** - The client interface where users interact with the agent
+2. **Azure Bot Service** - Intermediate service that handles authentication, schema validation, and message routing
+3. **Custom Proxy Application** - Web application that implements the bot activity protocol and communicates with Logic Apps via A2A (Application-to-Application)
+
+> [!NOTE]
+> Messages don't go directly from Teams to Logic Apps. Instead, they flow through Azure Bot Service, which then forwards them to your custom proxy application that communicates with Logic Apps.
+
+## Prerequisites
+
+- Azure subscription with Logic Apps Standard
+- Microsoft Teams with tenant admin permissions
+- Visual Studio or VS Code
+- Logic Apps agent already created and configured
+- GitHub repository access (sample code available)
+
+## Step 1: Create Logic App Agent
+
+Ensure you have a Logic Apps agent configured with:
+- Agent workflow with conversation handling
+- OAuth connection configured (if authentication required)
+- A2A (Application-to-Application) endpoint enabled
+
+> [!TIP]
+> Test your Logic Apps agent locally first using the Logic Apps designer before proceeding with Teams integration.
+
+## Step 2: Create Azure Bot Service
```

Comment:
can we add a screenshots (bot service and AAD app and show which of the values need to be extracted - such as client id, tenantid etc?